src/main/java/com/maxdemarzi/shortest/Service.java

```diff
@@ -325,6 +385,13 @@ private void streamShortestPathsUsingHandwrittenBFS(String centerEmail, List<Str
                         }
                     }
 
+                    if (level == 1) {
+                        // Pretend there are length 1 paths to the bibliograpahy entries
+                        for (Long bibId : bibliographyNodeIds) {
+                            pathsToNextLevel.putIfAbsent(bibId, Integer.valueOf(1));
```

Comment:
you don't actually want to put an `Integer` - this is a special collection type that maps primitives to primitives